rotkehlchen/accounting/accountant.py

```diff
@@ -82,16 +83,26 @@ def _process_skipping_exception(
         event = events[count]
         ts = event.get_timestamp()
         identifier = event.get_identifier()
-        self.msg_aggregator.add_error(
-            f'Skipping event with id {identifier} at '
-            f'{self.csvexporter.timestamp_to_date(ts)} '
-            f'during history processing due to {reason}: '
-            f'{str(exception)}. Check the logs for more details',
-        )
-        log.error(
-            f'Skipping event with id {identifier}  during history processing due to '
-            f'{reason}: {str(exception)}',
-        )
+
+        if isinstance(exception, NoPriceForGivenTimestamp):
```

Comment:
There is no need to have this if. This function is only called for that exception processing. Just move the `self.pots[0].cost_basis.missing_prices.append` part under the `except` processing for the `NoPriceForGivenTimestamp`.

The way you do it now you introduce an uneeded extra if check.